README.md

```diff
@@ -27,3 +27,30 @@ There are three ways to run `hdmi2usb-mode-switch`:
  1. Install the
  [udev-rules](https://github.com/litex-hub/litex-buildenv-udev)
  which sets the permissions of the unbind to the `video` group.
+
+## Building HDMI2USB firmware
```

Comment:
You can move this to a README.md in the same folder as `hdmi2usb/firmware/boot-dfu.ihex`

You should include what revision and version of the tooling was used to generate that `boot-dfu.ihex` file.